pages/curriculum.tsx

```diff
@@ -26,40 +26,26 @@ const Curriculum: React.FC = () => {
     const { lessons }: { lessons: Lesson[] } = data
     const sortedLessons: React.ReactElement[] = lessons
       .sort((a, b) => a.order - b.order)
-      .map((e: Lesson, idx: number) => (
-        <Card key={e.id} title={e.title}>
-          <img
-            src={`/curriculumAssets/lessonCoversSvg/js-${idx}-cover.svg`}
-            alt={`js${idx}-cover`}
-          />
-          <div>
-            <img
-              src="/curriculumAssets/icons/icon-lesson.svg"
-              alt="icon-lessons"
-            />
-            10 Lessons
-          </div>
-          <div>
-            <img
-              src="/curriculumAssets/icons/icon-challenge.svg"
-              alt="icon-challenge"
-            />
-            {e.challenges.length} Challenges
-          </div>
-          <div>
-            <img src="/curriculumAssets/icons/icon-time.svg" alt="icon-time" />2
-            Hours
-          </div>
-          <p>{e.description}</p>
-        </Card>
+      .map((e, idx) => (
+        <LessonCard
+          key={e.id}
+          coverImg={`js-${idx}-cover.svg`}
+          title={e.title}
+          lessonCount={10}
+          challengeCount={e.challenges.length}
+          hourCount="-"
+          description={e.description}
+        ></LessonCard>
       ))
-
     return (
       <Layout>
         <>
-          {sortedLessons}
-          <Card title="Progress"></Card>
-          <Card title="General Announcements"></Card>
+          <div className="row">
+            <div className="cards-container col-12 col-sm-9">
```

Comment:
why not keep it easy and do `col-9` here and `col-3` for the other div? Is there a benefit of making this so complicated?